.github/actions/publish_all_modules/action.yml

```diff
@@ -0,0 +1,35 @@
+name: 'Publish All Modules'
+description: 'Publishes all modules'
+inputs:
+  sonatype_usr:
+    description: 'Sonatype user'
+    required: true
+    default: ''
+  sonatype_pwd:
+    description: 'Sonatype password'
+    required: true
+    default: ''
+  signing_key_id:
+    description: 'Signing key id'
+    required: true
+    default: ''
+  signing_key_pwd:
```

Comment:
Let's spell out `sonatype_usr`, `sonatype_pwd`, and `signing_key_pwd` instead of abbreviating for clarity sake

Reply:
updated